src/main/resources/com/belleMusica/viewmodel/SearchUserResultsViewModel.hbs

```diff
@@ -5,7 +5,7 @@
         {{#each foundUsers}}
                 <a href="/{{id}}">
                     <div class="foundUserCard">
-                        <img class="profile_picture" src="{{this.profilePicture}}">
+                        <img class="profile_picture_search" src="{{this.profilePicture}}">
```

Comment:
Isn't better to add a id instead of changing the classname?

Reply:
nope, coz it's a list, id rather add an id for a unique element